server_installation/topics/operator/keycloak-client-cr.adoc

```diff
@@ -0,0 +1,67 @@
+
+=== KeycloakClient Custom Resource
+
+{project_operator} allows application developers to represent Keycloak Clients as Custom Resources:
+
+.`KeycloakClient` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakClient
+metadata:
+  name: <Keycloak Client name>
+  labels:
+    app: sso
+spec:
+  realmSelector:
+     matchLabels:
+      app: <matching labels for KeycloakRealm Custom Resource>
+  client:
+    # auto-generated if not supplied
+    #id: 123
+    clientId: client-secret
+    secret: client-secret
+    # ...
+    # other properties of Keycloak Client
+```
+
+TIP: Note, that `realmSelector` needs to match labels of an existing `KeycloakRealm` Custom Resource.
+
+{project_operator} synchronizes all the changes made to the Custom Resource with a running {project_name} instance. No manual changes via Keycloak Admin Console are allowed.
```

Comment:
Add a `NOTE:` annotation here, `NOTE: {project operator} .... No manual changes ... are allowed.`

Or maybe make it even `IMPORTANT:` if manual attempt is prohibited / might lead to error.

Reply:
I guess `NOTE` is fine.